src/openapi.yaml

```diff
@@ -1396,35 +1482,14 @@ paths:
               $ref: '#/components/headers/X-CSRFToken'
             X-Is-Form-Designer:
               $ref: '#/components/headers/X-Is-Form-Designer'
+            Content-Language:
+              $ref: '#/components/headers/Content-Language'
   /api/v2/forms/{uuid_or_slug}:
     get:
       operationId: forms_retrieve
-      description: |-
-        Manage forms.
-
-        Forms are collections of form steps, where each form step points to a formio.js
-        form definition. Multiple definitions are combined in logical steps to build a
-        multi-step/page form for end-users to fill out. Form definitions can be (and are)
-        re-used among different forms.
-
-        **Warning: the response data depends on user permissions**
-
-        Non-staff users receive a subset of the documented fields which are used
-        for internal form configuration. These fields are:
-
-        - `internalName`
-        - `registrationBackend`
-        - `registrationBackendOptions`
-        - `paymentBackend`
-        - `paymentBackendOptions`
-        - `product`
-        - `category`
-        - `isDeleted`
-        - `submissionConfirmationTemplate`
-        - `submissionsRemovalOptions`
-        - `confirmationEmailTemplate`
-        - `confirmationEmailOption`
-        - `displayMainWebsiteLink`
```

Comment:
Where did this go? I don't see this disappear in the code diff.

Reply:
oh I missed this, probably because of the explicit `retrieve` implementation and/or description that's now set rather than taken from the docstring. I'll sort it out!

Reply:
it's because I added `description` to `@extend_schema` which overrules the default (viewset docstring), I've incorporated it now.